src/Video.js

```diff
@@ -127,6 +127,16 @@ export const Initialize = async ({client, target, url}={}) => {
 
   params.playerParameters.clientOptions.client = client;
 
+  if(params.node) {
+    let nodes = await client.Nodes();
```

Comment:
is there a purpose to getting and setting `nodes` here? It doesn't seem to be used anywhere else and is unrelated to setting the client nodes

Reply:
Oh I was following the logic in elv-stream-sample in pushing the node to the fabric URIs. But I can remove